index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="Nav-bar">
+        <a href="#"> Solomon Afutu</a>
+        <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+      </nav>
+    </header>
+    <main>
+      <section class="header-main">
+        <h1>I’m Solomon Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h4>LET'S CONNECT</h4>
+        <ul>
+          <li>
+            <a href="https://twitter.com/SolomonAfutu"
+              ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+              ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://github.com/SolomonTheCoder"
+              ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href="">
+              <i class="fa fa-medium fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li>
+            <a href=""
+              ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
+            ></a>
+          </li>
+        </ul>
+      </section>
+      <section class="Cards">
+        <div class="Card1">
+          <img src="img/Snapshoot Portfolio.png" alt="" />
+          <div class="Card1-P">
+            <h4>Tonic</h4>
+            <div class="Card1-tech">
+              <div>
+                <p class="P1">Canopy</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P2">Back End Dev</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P3">2015</p>
+              </div>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer kindly fix the issue and let your code look like this
<img width="1086" alt="Screenshot 2023-03-23 at 16 05 41" src="https://user-images.githubusercontent.com/81444338/227384477-956f50b9-dab3-49f8-bcf3-d4a98aea1d57.png">